skbio/diversity/tests/test_driver.py

```diff
@@ -298,20 +298,20 @@ def test_invalid_input(self):
 
         # additional kwargs
         self.assertRaises(TypeError, beta_diversity, 'euclidean',
-                          [[0, 1, 3, 4], [0, 3, 12, 42]],
+                          [[0, 1, 3], [0, 3, 12]],
```

Comment:
I can update to use ``six.assertRaisesRegex``, but note that that wouldn't have caught this issue. The test was passing for the right reason.